<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterId</class>
 <widget class="QWidget" name="RegisterId">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>注册账号</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>15</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>72</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>账户名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>55</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>225</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#cancelButton {
	background-color: rgb(200, 200, 200);
	border-radius: 3px;
}

QPushButton#cancelButton:hover {
	background-color: rgb(170, 170, 170);
	border-radius: 3px;
}

QPushButton#cancelButton:pressed {
	background-color: rgb(150, 150, 150);
	border-radius: 3px;
}</string>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>225</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#logButton {
	background-color: rgb(200, 200, 200);
	border-radius: 3px;
}

QPushButton#logButton:hover {
	background-color: rgb(170, 170, 170);
	border-radius: 3px;
}

QPushButton#logButton:pressed {
	background-color: rgb(150, 150, 150);
	border-radius: 3px;
}</string>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>55</y>
     <width>72</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipEdit">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>135</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>95</y>
     <width>72</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>重复密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="repassEdit">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>95</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>135</y>
     <width>72</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>密码提示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bgLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>270</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>390</width>
     <height>270</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>390</width>
     <height>270</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-image: url(:/friend/bg.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>175</y>
     <width>72</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>175</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <zorder>bgLabel</zorder>
  <zorder>idEdit</zorder>
  <zorder>label</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>cancelButton</zorder>
  <zorder>logButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>repassEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>tipEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>nameEdit</zorder>
 </widget>
 <tabstops>
  <tabstop>idEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>repassEdit</tabstop>
  <tabstop>tipEdit</tabstop>
  <tabstop>logButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
